raven/1.5.1/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# base image
+FROM ubuntu:xenial
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="raven"
+LABEL software.version="1.5.1"
+LABEL description="Raven is a de novo genome assembler for long uncorrected reads."
+LABEL website="https://github.com/lbcb-sci/raven"
+LABEL license="https://github.com/lbcb-sci/raven/blob/master/LICENSE"
+LABEL maintainer="Erin Young"
+LABEL maintainer.email="eriny@utah.gov"
+
+# version to build with
+ARG RAVEN_VERSION=1.5.1
+ARG CMAKE_VERSION=3.21.0
+
+# install dependencies via apt-get or yum if using a centos or fedora base
+RUN apt-get update && apt-get install -y \
+ wget \
+ git \
+ clang \
+ zlib1g-dev \
+ gcc \
+ build-essential \
+ libssl-dev \
+ libcurl4-openssl-dev
```

Comment:
could you please add this to the end of the apt-get install layer ?` && apt-get autoclean && rm -rf /var/lib/apt/lists/* `

Reply:
done!